index.html

```diff
@@ -204,28 +221,104 @@ <h2 class="text-[32px] text-dark-blue font-bold mt-3 -mb-2">
-      <!--* end project one -->
+      <!--! end project one -->
 
-      <!--* project two -->
+      <!--!! project one -- md: number three -->
       <div
-        class="w-[100%] p-4 border border-light-grey-2 bg-white rounded-2xl flex flex-col justify-center items-center"
+        class="w-[100%] p-4 border md:order-3 md:h-[496px] md:gap-x-[2.9rem] border-light-grey-2 bg-white rounded-2xl flex flex-col md:flex-row justify-center items-center"
       >
         <!--* project screen shot -->
-        <div class="w-full h-[50%] bg-white">
+        <div class="w-full md:w-[50%] h-[50%] md:h-full bg-white">
           <img
             class="w-full h-full object-contain"
-            src="./img/projects/project-2.png"
+            src="./img/desktop/desktop-project-3.jpg"
             alt="project-1"
           />
         </div>
         <!--* project info -->
-        <div class="flex flex-col justify-around items-start gap-y-4">
-          <h2 class="text-[32px] text-dark-blue font-bold mt-3 -mb-2">
+        <div
+          class="flex md:w-[50%] md:h-full flex-col justify-around md:justify-start items-start gap-y-4"
+        >
+          <h2
+            class="text-[32px] md:text-[40px] text-dark-blue font-bold mt-3 -mb-2"
+          >
+            Tonic
+          </h2>
+          <ul
+            class="list-none flex md:text-lg justify-between items-center w-[240px]"
+          >
+            <li class="text-light-dark-blue text-small font-semibold">
+              CANOPY
+            </li>
+            <li class="h-2 w-2 bg-light-grey rounded-full"></li>
+            <li class="text-light-grey-4 font-semibold text-small">
+              Back End Dev
+            </li>
+            <li class="h-2 w-2 bg-light-grey rounded-full"></li>
+            <li class="text-light-grey-4 font-semibold text-small">
+              2015
+            </li>
+          </ul>
+          <p class="text-light-teal md:text-base text-[15px]">
+            A daily selection of privately personalized reads; no
+            accounts or sign-ups required.
+          </p>
+          <ul class="w-[230px] flex justify-between items-center">
+            <li>
+              <a
+                href="#"
+                class="text-light-blue text-xs px-3 py-1 bg-faded-blue rounded-lg"
+                >html</a
+              >
+            </li>
+            <li>
+              <a
+                href="#"
+                class="text-light-blue text-xs px-3 py-1 bg-faded-blue rounded-lg"
+                >css</a
+              >
+            </li>
+            <li>
+              <a
+                href="#"
+                class="text-light-blue text-xs px-3 py-1 bg-faded-blue rounded-lg"
+                >javascript</a
+              >
+            </li>
+          </ul>
+          <a
+            href="#"
+            class="inline-block text-primary-blue p-4 text-[17px] border border-light-blue rounded-lg mb-[14px]"
+            >see project</a
+          >
```

Comment:
- [ ] It would be better also to add the button interaction on all your buttons and match those of Figma design for the best user experience 😂 

   You may follow this[ link](https://css-tricks.com/a-complete-guide-to-links-and-buttons/) for A Complete Guide to Links and Buttons interaction 

**Figma buttons**
![image](https://user-images.githubusercontent.com/102757126/206837835-c199ed93-2b0d-45a9-b0dc-54547b45c701.png)